### ISSUE TITLE
Catalog Management Improvements

### ISSUE THREAD

**Issue by JeffOdegard** [cyberperspectives/sagacity#24]
On the catalog management page add buttons for download, offline and update

Download does an download only of all required files for offline installation, equivalent to update_db.php --cpe --nvd --stig --sunset --do, then zips up the resulting files and offers the .zip file to be saved wherever they choose.  (Contains the tmp folder files and directories needed for offline install)

Offline opens a dialogue to upload the .zip file described above, then unzips it and does the equivalent of update_db.php --cpe --nvd --stig --sunset --po

Update would simply run update_db.php --cpe --nvd --stig --sunset

**Comment by godsgood33** [comment]
How about moving this to the settings page under the table where the counts are displayed?  Maybe even edit the table to include a checkbox for each type that they'd like to import.

**Comment by JeffOdegard** [comment]
This is a catalog function.  I think it belongs under Catalog Management.

Maybe if you click the download, offline or update buttons it opens a dialog with the checkboxes for each data type.

**Comment by godsgood33** [comment]
I agree, in general, that it is a catalog function, but the Catalog Management page only displays the STIGs.  We can look at revising that.  Maybe having a dropdown above the table where they select the type (CPE, CVE, STIG, NASL, etc), and it will fetch the data and generate the table?  Then next to that dropdown there would be an option to update with a series of checkbox for types and action (do / po)?  If you like this, I can try to do up a prototype to screenshot.
